pagefaultgames/pokerogue
# [QoL Balance] Shiny Chance from 1/2048 -> 1/1024, shiny charm adjustment

## What are the changes?
Changes the base Shiny Chance after discussion from 1/2048 -> 1/1024

## Why am I doing these changes?
Shinies are a part of the overall progression of the game for most circumstances. 
The *INTENDED* affect is halving the highest end of odds, and making sure the Shiny Charm still ends at 1/32 at 4 stacked

## What did change?
Changed one 32 to a 64 for shiny chance, and a 2 to a 1 for shiny charms as of now.

### Screenshots/Videos
N/A

## How to test the changes?
Merge the PR with an offline version, and/or double check the code to make sure the right lines are adjusted.

## Checklist
